### ISSUE TITLE
Can't focus TextField inside a TabBarView using VoiceOver gestures 

### ISSUE THREAD

**Issue by kpsroka** [flutter/flutter#45394]
## Steps to Reproduce

See [this gist](https://gist.github.com/kpsroka/caa9eaf0d178f36178025ca7ae466b85) with bug reproduction.

1. Create a widget wrapping TextField within a TabBarView.
2. Install your app on an iOS device.
3. Enable VoiceOver.
4. Use a11y gestures to navigate to the TabBarView content. You should hear a TextField announcement with "double tap to edit" spoken out.
5. Double tap.

Observed bug: Nothing changes - text field is not activated, focus doesn't change. You are also unable to focus on the text field itself.

**Target Platform: iOS**
**Target OS version/browser: 13.1.2**
**Devices: iPhone XR**

The bug occurs in both debug and release modes.

## Logs

[logs.txt](https://github.com/flutter/flutter/files/3879684/logs.txt)

```
$ flutter analyze
Analyzing tab_bar_view_semantics...                                     
No issues found! (ran in 4.4s)
```

```
$ flutter doctor -v
[✓] Flutter (Channel stable, v1.9.1+hotfix.6, on Mac OS X 10.14.6 18G103, locale en-PL)
    • Flutter version 1.9.1+hotfix.6 at /Users/kpsroka/Library/flutter
    • Framework revision 68587a0916 (10 weeks ago), 2019-09-13 19:46:58 -0700
    • Engine revision b863200c37
    • Dart version 2.5.0

 
[!] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/kpsroka/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)
    ! Some Android licenses not accepted.  To resolve this, run: flutter doctor --android-licenses

[✓] Xcode - develop for iOS and macOS (Xcode 11.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.1, Build version 11A1027
    • CocoaPods version 1.8.4

[✓] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 40.0.2
    • Dart plugin version 191.8423
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)

[✓] VS Code (version 1.40.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.6.0

[✓] Connected device (1 available)
    • Krzysztof’s iPhone • 00008020-000D544E1AF9002E • ios • iOS 13.1.2
```

**Comment by perclasson** [comment]
I can reproduce this bug.

For a workaround if you put a `MergeSemantics` around the `TextField`, the keyboard will open when double tapping with VoiceOver.

I have yet to find a root cause, the `debugDumpSemanticsTree` does not show anything obvious.

**Comment by darshankawar** [comment]
Hi @kpsroka,
Can you please confirm if you still experience this issue on latest Flutter channel (eg: dev, master) ?

**Comment by darshankawar** [comment]
Without additional information we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.
